docs/tutorial/index.md

```diff
@@ -317,245 +387,239 @@ Removing a unit from the application scales down the replicas.
 
-Unit           Workload  Agent  Machine  Public address  Ports  Message
-postgresql/0*  active    idle   0        10.89.49.129           
-postgresql/1   active    idle   1        10.89.49.197           
-
-Machine  State    Address       Inst id        Series  AZ  Message
-0        started  10.89.49.129  juju-a8a31d-0  jammy       Running
-1        started  10.89.49.197  juju-a8a31d-1  jammy       Running
+Unit           Workload  Agent      Machine  Public address    Ports     Message   
+postgresql/0*  active    idle       0        10.26.224.154     5432/tcp                          
+postgresql/1   active    executing  1        10.26.224.142     
+  5432/tcp   
+                                                                                                                                              
+Machine  State    Address         Inst id        Base          AZ  Message
+0        started  10.26.224.154   juju-1c143d-0  ubuntu@24.04      Running
+1        started  10.26.224.142   juju-1c143d-1  ubuntu@24.04      Running
 ```
 
-## Manage passwords
-
-When we accessed PostgreSQL earlier in this tutorial, we needed to use a password manually. Passwords help to secure our database and are essential for security. Over time, it is a good practice to change the password frequently. 
-
-The operator's password can be retrieved by running the `get-password` action on the Charmed PostgreSQL VM application:
+## Integrate with other applications
 
-```text
-juju run postgresql/leader get-password
-```
-Running the command should output something like:
+[Integrations](https://documentation.ubuntu.com/juju/3.6/reference/relation/), also known as "relations", are the easiest way to create a user for PostgreSQL in Charmed PostgreSQL. 
 
-```yaml
-unit-postgresql-0:
-  UnitId: postgresql/0
-  id: "14"
-  results:
-    operator-password: eItxBiOYeMf7seSv
-  status: completed
-  timing:
-    completed: 2023-03-20 09:17:51 +0000 UTC
-    enqueued: 2023-03-20 09:17:49 +0000 UTC
-    started: 2023-03-20 09:17:50 +0000 UTC
-```
+Integrations automatically create a username, password, and database for the desired user/application. The best practice is to connect to PostgreSQL via a specific user rather than the admin user, like we did earlier with the `operator` user.
 
-You can change the operator's password to a new random password by entering:
+In this tutorial, we will relate to the [data integrator charm](https://charmhub.io/data-integrator). This is a bare-bones charm that allows for central management of database users. It automatically provides credentials and endpoints that are needed to connect with a charmed database application.
 
-```text
-juju run postgresql/leader set-password
-```
+To deploy `data-integrator` and associate it to a new database called `mydb-integrator`:
 
-Running the command should output something like:
+```{terminal}
+:input: juju deploy data-integrator --config database-name=mydb-integrator
```

Comment:
There's now a noble based data-int in stable that can be deployed with `--base ubuntu@24.04`.